docs/debugging.md

```diff
@@ -6,47 +6,41 @@ description: Debug your Next.js app.
 
 ## Step 1: Start Next.js in debug mode
 
-Next.js being a Node.js application, all we have to do is to pass down the [`--inspect`](https://nodejs.org/api/cli.html#cli_node_options_options) flag to the underlying Node.js process for it to start in debugger mode.
+Next.js being a Node.js application, all we have to do is to pass down the [`--inspect`](https://nodejs.org/api/cli.html#cli_node_options_options) flag to the underlying Node.js process for it to start in debug mode.
 
 First, start Next.js with the inspect flag:
 
 ```bash
-NODE_OPTIONS='--inspect' next
+NODE_OPTIONS='--inspect' next dev
 ```
 
-If you're using `npm run dev` or `yarn run dev` from the [getting started](/docs/getting-started) and `create-next-app` then you could also change your `package.json` this way:
+If you're using `npm run dev` or `yarn dev` (See: [Getting Started](/docs/getting-started)) then you should update the `dev` script on your `package.json`:
 
 ```json
-"scripts": {
-  "dev": "NODE_OPTIONS='--inspect' next"
-}
+"dev": "NODE_OPTIONS='--inspect' next dev"
 ```
 
-So that in development mode, you're always able to debug your application without having to restart your process.
-
 The result of launching Next.js with the inspect flag looks like this:
 
 ```bash
 Debugger listening on ws://127.0.0.1:9229/0cf90313-350d-4466-a748-cd60f4e47c95
 For help, see: https://nodejs.org/en/docs/inspector
-[ wait ]  starting the development server ...
-[ info ]  waiting on http://localhost:3000 ...
-[ ready ] compiled successfully - ready on http://localhost:3000
+ready - started server on http://localhost:3000
 ```
 
-_Be aware that using `NODE_OPTIONS='--inspect' npm run dev` or `NODE_OPTIONS='--inspect' yarn run dev` won't work. This would try to start multiple debuggers on the same port: one for the npm/yarn process and one for Next.js. You would then get an error like `Starting inspector on 127.0.0.1:9229 failed: address already in use` in your console_.
+> Be aware that using `NODE_OPTIONS='--inspect' npm run dev` or `NODE_OPTIONS='--inspect' yarn dev` won't work. This would try to start multiple debuggers on the same port: one for the npm/yarn process and one for Next.js. You would then get an error like `Starting inspector on 127.0.0.1:9229 failed: address already in use` in your console.
 
 ## Step 2: Connect to the debugger
 
 ### Using the Chrome DevTools
 
-Open a new tab in Google Chrome and go to `chrome://inspect`, you should see your Next.js application inside the "Remote Target" section. Now click "inspect" to open a screen that will be your debugging environement from now on.
+Once you open a new tab in Google Chrome and go to `chrome://inspect`, you should see your Next.js application inside the "Remote Target" section. Now click "inspect" to open a screen that will be your debugging environement from now on.
 
 ### Using the Visual Studio Code debug panel
```

Comment:
I believe that view is called `Debugger`, which includes the Debug Panel and Debug Console.
```suggestion
### Using the Debugger in Visual Studio Code
```

Reply:
@lfades I think it’s actually called the Run View: https://code.visualstudio.com/docs/editor/debugging#_run-view But I think "debugger" is fine.